loops.js

```diff
@@ -46,13 +55,34 @@
 // ==========================================
 
 
-
+for (let i = 0; i <= 9; i++) {
+    if (i < 3) {
+        console.log(i);
+    } else if (i >= 3 && i < 6) {
```

Comment:
```suggestion
    } else if (i < 6) {
```
Omdat de eerste if al niet waar was, moet het getal wel groter of gelijk zijn aan 3